index.bs

```diff
@@ -2805,9 +3221,15 @@ that overloaded operations, legacy callers and constructors can be
 specified to take, and in order to describe these restrictions,
 the notion of an <em>effective overload set</em> is used.
 
-[=Operations=] and [=legacy callers=]
-must not be overloaded across [=interface=]
-and [=partial interface=] definitions.
+[=Operations=] must not be overloaded across
+[=interface=],
+[=partial interface=],
+[=mixin=], and
+[=partial mixin=]
+definitions.
+ 
+Similarly, [=legacy callers=] must not be overloaded across
```

Comment:
@tobie You might want rebase. Legacy callers are gone, and no one is missing them.

Reply:
Oh. Good point.